packages/react-native-sdk/src/constants/A11yLabels.ts

```diff
@@ -7,14 +7,22 @@ export enum A11yIcons {
 export enum A11yComponents {
   PARTICIPANT_AVATAR = 'participant-avatar',
   CALL_PARTICIPANTS_LIST = 'call-participants-list',
+  CALL_PARTICIPANTS_SPOTLIGHT_VIEW = 'call-participants-spotlight-view',
+  CALL_PARTICIPANTS_GRID_VIEW = 'call-participants-grid-view',
   LOCAL_PARTICIPANT = 'local-participant',
+  LOCAL_PARTICIPANT_FULLSCREEN = 'local-participants-fullscreen',
   PARTICIPANT_MEDIA_STREAM = 'participant-media-stream',
+  PARTICIPANTS_INFO_VIEW = 'participants-info-view',
+  PARTICIPANT_VIEW_SCREEN_SHARING = 'participant-view-screen-sharing',
+  REACTIONS_MODAL = 'reactions-modal',
+  CHAT_UNREAD_BADGE_COUNT_INDICATOR = 'chat-unread-badge-count-indicator',
 }
 
 export enum A11yButtons {
   PARTICIPANTS_INFO = 'participants-info-button',
   EXIT_PARTICIPANTS_INFO = 'exit-participants-info-button',
   HANG_UP_CALL = 'hang-up-call-button',
+  REACTION_BUTTON = 'call-controls-reaction-button',
```

Comment:
```suggestion
  REACTION = 'call-controls-reaction-button',
```

No need for postfix _BUTTON here we already use it by calling A11yButtons.REACTION